src/render-dom.js

```diff
@@ -196,11 +197,17 @@ function applyToDOM(container, definitionFn, props = null) {
   let interactions = makeInteractions(rootElem$);
   let output = definitionFn(interactions, props);
   let {vtree$, customEvents} = digestDefinitionFnOutput(output);
-  let subscription = rootElem$.connect();
+  let connection = rootElem$.connect();
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).